pkg/cmd/flags.go

```diff
@@ -94,6 +94,11 @@ func AddCloudFlag(cmd *cobra.Command) {
 	RegisterFlagCompletionFunc(cmd, "cloud", func(_ *cobra.Command, _ []string) []string { return kafka.Clouds })
 }
 
+func AddPluginCloudFlag(cmd *cobra.Command) {
+	cmd.Flags().String("cloud", "", fmt.Sprintf("Cloud provider of custom plugin. Supported value %s. (default \"aws\").", utils.ArrayToCommaDelimitedString([]string{"aws", "azure"}, "or")))
```

Comment:
```suggestion
	cmd.Flags().String("cloud", "aws", fmt.Sprintf("Set cloud provider of custom plugin as %s.", utils.ArrayToCommaDelimitedString([]string{"aws", "azure"}, "or")))
```

Reply:
Nit: Define the slice as a variable so we only need to edit the list of accepted cloud providers once

Reply:
Just curious, what's the reason for defaulting to AWS instead of making this a required field? Don't we try to treat all cloud providers equally as a company?

Reply:
while we create api change and talk to api team, they recommend to use as optional. If use as optional this will not be a breaking change. 
Our team is working on supporting azure Q1 (not announce now but at the end of Q1), and will start to work on supporting gcp in Q2. So we  not want to give user the gcp option now.